cli/cmd/install.go

```diff
@@ -69,16 +52,13 @@ var (
 		"templates/serviceprofile.yaml",
 	}
 
-	templatesConfigStage = []string{
+	templatesControlPlaneStage = []string{
```

Comment:
nit: Since we don't have stages anymore, this could be renamed to `templatesControlPlane`... but yes just a nit.